includes/Migrations/Add_Media_Source_Gif_Conversion.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Class Add_Media_Source_Gif_Conversion
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+namespace Google\Web_Stories\Migrations;
+
+use Google\Web_Stories\Media;
+
+/**
+ * Class Add_Media_Source_Gif_Conversion
+ *
+ * @package Google\Web_Stories\Migrations
+ */
+class Add_Media_Source_Gif_Conversion extends Migrate_Base {
+
+	/**
+	 * Add the gif-conversion term, to make sure it exists.
+	 *
+	 * @since 1.4.0
+	 *
+	 * @return void
+	 */
+	public function migrate() {
+		wp_insert_term( 'gif-conversion', Media\Media::STORY_MEDIA_TAXONOMY );
+		wp_insert_term( 'source-image', Media\Media::STORY_MEDIA_TAXONOMY );
```

Comment:
Nit: technically the name of the class is now incorrect.

Reply:
Broken into two files.